src/pages/VaultList/components/Vault/Vault_View.tsx

```diff
@@ -46,8 +46,8 @@ const VaultView = ({
             <h1 className="text-3xl mb-2">Deposit / Withdraw</h1>
             <span>Shares: {userBalance ?? <Skeleton />}</span>
 
-            <label className="flex whitespace-nowrap">
-              <span className="text-left mr-2 mt-2">Amount:</span>
+            <span className="flex flex-col text-left mr-2 mt-2">Amount</span>
```

Comment:
why flex flex-col and text-left? all text is left aligned by default

Reply:
Removed the text-left but needed the flex-col :)